dev/tools/lib/roll_dev.dart

```diff
@@ -165,23 +167,26 @@ void main(List<String> args) {
 }
 
 String getFullTag() {
+  const String glob = '*.*.*-*.*.pre';
   return getGitOutput(
-    'describe --match *.*.*-dev.*.* --first-parent --long --tags',
+    'describe --match $glob --first-parent --long --tags',
     'obtain last released version number',
   );
 }
 
 Match parseFullTag(String version) {
-  final RegExp versionPattern = RegExp(r'^([0-9]+)\.([0-9]+)\.([0-9]+)-dev\.([0-9]+)\.([0-9]+)-([0-9]+)-g([a-f0-9]+)$');
+  final RegExp versionPattern = RegExp(
```

Comment:
Can you add a comment with an example of a tag that should match?

Reply:
done